src/Input/AutoComplete/AutoComplete.tsx

```diff
@@ -219,6 +217,8 @@ class AutoComplete extends React.Component<Props, State> {
       onChange,
       small,
       removeFloatingLabel,
+      children, // eslint-disable-line @typescript-eslint/no-unused-vars
+      value, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
🤷‍♂

Reply:
I was feeling 🤷‍♂too when I did this haha.

If we just delete this too variable, it will be included in `defaultProps` below them which will be passed to `AutoCompleteInput`, and thus it causes the bugs like overwrite the `value={selectedValue}`, `input tag element should not have a children` something like this.

I was thinking another soultion as below, but I just think the current way is simpler.
```
const autoCompleteInputProps = Object.assign({}, defaultProps);
delete autoCompleteInputProps.value;
delete autoCompleteInputProps.children;
<AutoCompleteInput
 // ...
 {...autoCompleteInputProps}
/>
```

Wdyt?

Reply:
This is not related to this PR so just leave it as is.

Reply:
I am thinking do we really need to extend the type `ComponentPropsWithoutRef` for `Props`